### ISSUE TITLE
Dashboard to generate a working manifest?

### ISSUE THREAD

**Issue by aabedraba** [kubeshop/kusk-gateway#429]
**Describe the enhancement you'd like to see**
Currently when an API is updated/created through the dashboard, the user loses sync between the configuration that is deployed to the cluster and the configuration saved in Github for example. This means that the users loses "the source of truth" when they use the dashboard... 

Suggested solution is that the dashboard should not deploy the API, but just generate the openapi.yaml/openapi.json file needed to configure Kusk Gateway. It is then the developer team's job to keep that file in sync.

**Additional context**
This is a solution that I observed from Krakend, where they have a public page to design their Gateway's configuration https://designer.krakend.io/#!/

**Comment by olensmar** [comment]
good point @aabedraba - the initial idea of this functionality was to make it easy for users to "try out" KGW with an existing OpenAPI definition - you're totally right that it moves away from the intended automated/design-first workflow - which is perhaps something we should call out in the UI in line with your suggestion. I'm also wondering if we could create some kind of "automation wizard" that sets up automatic deployment from a github repo (for example) - without requiring the user to install Argo/Flux/etc!?
